src/file-picker/src/FilePicker.js

```diff
@@ -157,4 +164,17 @@ export default class FilePicker extends PureComponent {
   handleButtonClick = () => {
     this.fileInput.click()
   }
+
+  handleBlur = e => {
+    const { onBlur } = this.props
+    const { files } = this.state
+
+    // Redux-form derives field value from bubbled onBlur event, which is fake here
+    if (e && e.stopPropagation) e.stopPropagation()
```

Comment:
I'm concerned this change might be a breaking one.

Why must we stop the event from propagating?

Reply:
If we don't, the event bubbles up to redux-form Field, which derives field value from `event.target.files`: https://github.com/erikras/redux-form/blob/master/src/events/getValue.js#L32
The problem with that event is that it's fake, not real `<input type="file/>` event.
We could try to augment that with `target.files` property, that'd also fix that.

Reply:
I am wondering why we wouldn't handle that in consumers since it's a dependency of `redux-form`. e.g. wrap your field with `onBlur` handling that does the stop propagation.

Otherwise this could be a breaking change for some consumers and would have to wait for v5.

Reply:
That works, but that is unknown hack for now.

```js
<Field name="file" component={({props}) => (
<Box onBlur={e => e.stopPropagation()}>
    <FilePicker
        name={props.input.name}
        onChange={props.input.onChange}
        onFocus={props.input.onFocus}
        height={40} />
</Box>)}/>
```

What about augmenting `event.target.files`?